### ISSUE TITLE
[Feature request] Provide a regex to match classes, with sensible defaults

### ISSUE THREAD

**Issue by jgoux** [francoismassart/eslint-plugin-tailwindcss#7]
Hello,

**Is your feature request related to a problem? Please describe.**
I'm trying out this plugin to sort my tailwind classes in the context of twin.macro. The classes are declared using the `tw` tagged template string and it seems that it's not detected by the plugin.

**Describe the solution you'd like**
I'd like it to be detected automatically as it's a broad adopted convention in CSS in JS.
For more flexibility you could provide a regex we can pass to the configuration of the plugin to match the classes according to what people use. Then the className and tw would just be part of the default regex.

This is what the Tailwind VSCode extension is doing : 
```js
{
  "tailwindCSS.experimental.classRegex": [
      "tw`([^`]*)", // tw`...`
      "tw=\"([^\"]*)", // <div tw="..." />
      "tw={\"([^\"}]*)", // <div tw={"..."} />
      "tw\\.\\w+`([^`]*)", // tw.xxx`...`
      "tw\\(.*?\\)`([^`]*)" // tw(Component)`...`
  ],
}
```

**Comment by francoismassart** [comment]
Hello @jgoux,
sorry for the late reply, I was away from home & I'm still on holiday for few days.
I've never use [twin.macro](https://github.com/ben-rogerson/twin.macro), I did read their README and it can get quite complex fast due to the huge number of supported syntaxes.

I can't promise I will be able to support your feature request soon but the project is open to contributions so don't hesitate to jump in.

**Comment by francoismassart** [comment]
@jgoux, I added [few failing tests on a dedicated branch](https://github.com/francoismassart/eslint-plugin-tailwindcss/commit/9c356f4b50b90548241097767b74bf8c33e6f5ca), can you have a look at it and let me know if you see something wrong/missing ?

Also, I did not found any example for this snippet:
`<div tw={"..."} />`

**Comment by francoismassart** [comment]
I don't think I'll implement this myself, but if someone is up to it, contributions are welcome.
